src/code-editor/preferences-modal.tsx

```diff
@@ -97,6 +100,9 @@ export default (props: PreferencesModalProps) => {
               onChange={onThemeSelected}
               options={themeOptions}
               filteringType="auto"
```

Comment:
Did we consider removing filtering mode instead (to make the default setup automatically compliant)?

Since [our recommendation](https://cloudscape.design/components/code-editor/?tabId=api) is to limit allowed themes using `themes` prop, this list is unlikely to grow large

Reply:
it feels like this would probably be a breaking change. At the very least I'd want to give customers the option to re-enable filtering, in which case the properties would be needed anyway, but I could see an argument for adding that but turning it off by default, to improve the default accessibility via a slightly-breaking change (with re-opt-in option)

Reply:
Your call. But I feel we are missing a good simplification opportunity